stable/artifactory-ha/README.md

```diff
@@ -729,6 +729,20 @@ This can be done with the following parameters
 --set database.password=${DB_PASSWORD} \
 ...
 ```
+```bash
```

Comment:
@rahulsadanandan  can you add a heading - "Configuring Artifactory with external Oracle database" - Do same for artifactory chart

Reply:
Corrected heading and added note .